addon/components/paper-item.js

```diff
@@ -82,10 +82,14 @@ export default Component.extend(RippleMixin, ProxyMixin, {
         let el = component.$();
         set(this, 'mouseActive', false);
         el.on('mousedown', ()=> {
-          set(this, 'mouseActive', true);
-          run.later(()=> {
-            set(this, 'mouseActive', false);
-          }, 100);
+          if (!get(component, 'isDestroyed') && !get(component, 'isDestroying')) {
```

Comment:
Now that you mention it, @xomaczar @miguelcobain Is it the intended behavior that the ripple extend from the clicked component, across the entire paper item? 

In the case of clicking on a button it doesn't seem to make much sense?